tf_files/aws/commons/variables.tf

```diff
@@ -402,4 +402,19 @@ variable "network_expansion" {
 
 variable "rds_instance_storage_encrypted"{
   default = true
-}
\ No newline at end of file
+}
+
+variable "fence_max_allocated_storage" {
+  description = "Maximum allocated storage for autosacaling"
+  default     = 0
+}
+
+variable "sheepdog_max_allocated_storage" {
+  description = "Maximum allocated storage for autosacaling"
+  default     = 0
+}
+
+variable "indexd_max_allocated_storage" {
+  description = "Maximum allocated storage for autosacaling"
+  default     = 0
```

Comment:
This is awesome!
What does `default = 0` mean?  Does that mean unrestricted, or don't autoscale?

Reply:
means autoscaling is disabled